04-cs-fundamentals/classroom/recursion.md

```diff
@@ -0,0 +1,343 @@
+# Introduction to Recursion
+
+## Learning Goals
+
+By the end of this lesson you be able to:
+
+- Explain recursion
+- Trace through a recursive function call and understand time and space comlexities
+- Author a recursive algorithm
+
+## Video Lesson
+
+- [Video Lesson]() - Coming Soon
+- [Slides](https://docs.google.com/presentation/d/1SfpBdIgs38fj45WiwlFe6x0s0F44WqxPNHSizafzhwY) - Coming Soon
```

Comment:
These have a link but also have a "Coming Soon".

Reply:
Mostly a marker for me to change when the video lesson is recorded.